templates/dnsupdate-rfc2136/0/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+rfc2136dns:
+  image: rancher/external-dns:v0.4.2
+  expose:
+   - 1000
+  command:
+  - -provider
+  - RFC2136
+  environment:
+    DNS_HOST: ${DNS_HOST}
+    DNS_PORT: ${DNS_PORT}
+    TSIG_KEYNAME: ${TSIG_KEYNAME}
+    TSIG_KEY: ${TSIG_KEY}
```

Comment:
@digitalLumberjack Can you please update the environment variables to match those in the rfc2136 provider?
`DNS_HOST` => `RFC2136_HOST`
`DNS_PORT` => `RFC2136_PORT`
`TSIG_KEYNAME` => `RFC2136_TSIG_KEYNAME`
`TSIG_KEY` => `RFC2136_TSIG_SECRET`

See https://github.com/rancher/external-dns/blob/master/providers/rfc2136/rfc2136.go#L26-L41